Cargo.toml

```diff
@@ -47,7 +47,6 @@ members = [
     "crates/rpc/rpc-testing-util/",
     "crates/rpc/rpc-types/",
     "crates/rpc/rpc-types-compat/",
-    "crates/rpc/rpc-layer",
```

Comment:
we like this alphabetically sorted, please put it back here and remove it from down below